Library/Homebrew/extend/os/mac/dependency_collector.rb

```diff
@@ -23,4 +23,8 @@ def ld64_dep_if_needed(*)
     return if MacOS.version > :tiger
     LD64Dependency.new
   end
+
+  def zip_dep_if_needed(*); end
+
+  def bzip2_dep_if_needed(*); end
```

Comment:
👍

Reply:
Name the argument `tags` like `subversion_dep_if_needed` above.

Reply:
Also: put these below `xz_dep_if_needed`